dim_checks/mozdata/firefox_ios/active_users_aggregates/dim_checks.yaml

```diff
@@ -0,0 +1,51 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  slack_alerts:
+    enabled: true
+    notification_level: INFO
+    notify_channel: true
+    notify:
+      channels:
+        - kpi-monitoring-dim
+  tier: tier_1
+  partition_field: submission_date
+  dim_tests:
+    - type: custom_compare_metrics
+      title: DAU validation - Firefox iOS
+      description: Validating that DAU sum in Active Users Aggregates is the same as unique client_id count in the live tables (all channels).
+      params:
+        condition: dau_sum = distinct_client_count
+        metric_1:
+          name: dau_sum
+          query: |
+            SELECT SUM(dau),
+            FROM `{{ project_id }}.{{ dataset }}.{{ table }}`
+            WHERE submission_date = "{{ params.partition }}"
+        metric_2:
+          name: distinct_client_count
+          query: |
+            WITH base AS (
+              SELECT
+                "release" AS channel,
```

Comment:
Even though we don't retrieve the channel in the result, I find it very useful to have it for context!